package.json

```diff
@@ -5,14 +5,6 @@
   "private": true,
   "main": "",
   "scripts": {
-    "mongoPull": "sudo docker pull mongo",
-    "premongoRun" : "sudo mkdir -p ./data",
-    "mongoRun": "sudo docker run -d -v data:/data/db -p 27017:27017 --name mongo mongo",
-    "mongoStop": "sudo docker stop mongo && sudo docker rm mongo",
-    "amqpPull": "sudo docker pull semanticbus/rabbitmq-stomp",
-    "amqpStop": "sudo docker stop rabbitmq && sudo docker rm rabbitmq",
-    "amqpRun": "sudo docker run -d -p 15672:15672 -p 15674:15674 -p 5672:5672 --name rabbitmq semanticbus/rabbitmq-stomp",
-    "amqpRun2": "sudo docker run -d -p 15674:15674 -P --name rabbitmq semanticbus/rabbitmq-stomp",
```

Comment:
I removed all these commands because `docker-compose` take care of them.
However, if I did not see something important here, please let me know.

Reply:
an we still those command combined to docker compose please. We will clean after some well usage time.